cartoframes/auth/credentials.py

```diff
@@ -194,3 +202,23 @@ def delete(self, config_file=None):
             warnings.warn('Credentials at {} successfully removed.'.format(path_to_remove))
         except OSError:
             warnings.warn('No credential file found at {}.'.format(path_to_remove))
+
+    def get_do_token(self):
+        do_token_manager = DoTokenManager(self.get_api_key_auth_client())
+        token = do_token_manager.get()
+        if not token:
+            raise CartoException('Authentication error: do you have permissions to access Data Observatory data?')
```

Comment:
I'm proposing this change to avoid confusion with DO v1 which has it's own permission.

```suggestion
            raise CartoException('Authentication error: do you have permissions to access Data Observatory v2?')
```